04-vector-raster-integration-advanced.Rmd

```diff
@@ -159,29 +156,37 @@ dev.off()
 ```
 
 The graphic above illustrates the extent of several of the spatial layers that 
-we have worked with in this vector data lesson series including:
+we have worked with in this vector data lesson series:
 
-* Area of interest (AOI).
-* A roads/trails.
-* And plot locations.
-* A canopy height model (CHM) in GeoTIFF format.
+* Area of interest (AOI) -- blue
+* Roads and trails -- purple
+* Vegetation plot locations -- black
 
-However, sometimes we have a raster file that is must larger than our study 
-area or area of interest. It is often more efficient to `crop` the raster to the 
-extent of our study area, to keep file sizes smaller as we process our data. 
+and a raster file, that we will introduce this lesson: 
 
+* A canopy height model (CHM) in GeoTIFF format -- green
 
-## Import Data
-We will use four vector shapefiles with data on (field site boundary,
-roads/trails, tower location, and study plot locations) and one raster GeoTIFF
-file, a Canopy Height Model for the Harvard Forest, Massachusetts. 
+The raster files that we have to work with will rarely match the desired vector 
```

Comment:
I find the original text confusing because it starts with "However", contrasting the need for crop by extent for very large files to the previous 4 files, yet we continue to work with those for the rest of the lesson.  Additionally this being after the plot of the files doesn't flow well. 

Does this work better:  
plot description moved to before plot
plot of vector/raster extents

Frequent use cases of cropping a raster file include reducing file size and
creating maps.   Sometimes we have a raster file that is much larger than our study area or area
of interest. It is often most efficient to crop the raster to the extent of our
study area to keep reduce file sizes as we process our data.
Cropping a raster can also be useful when creating pretty maps so that the
raster layer matches the extent of the desired vector layers.